src/spatialdata_plot/pl/basic.py

```diff
@@ -291,17 +302,28 @@ def render_points(
         elements : list[str] | str | None, optional
             The name(s) of the points element(s) to render. If `None`, all points
             elements in the `SpatialData` object will be used.
-        color : Colorlike | str | None, optional
-            Can either be a color-like or a key in :attr:`sdata.table.obs`. The latter
-            can be used to color by categorical or continuous variables.
+        color : list[str | None] | str | None
+            Can either be string(s) representing a color-like or key(s) in :attr:`sdata.table.obs`. The latter
```

Comment:
I don't think it can be stuff in adata.obs, as points are not annotated by tables

Reply:
also below, is this correct? I don't think so,I think it was copy pasted?

Reply:
hehe points now can be annotated by tables and I agree on tests, still have to write them. In general also a lot of the notebooks have plots that have no equivalent tests here so before a lot of them were not properly showing either. However, for release it was pushed. The logic of this library is quite complicated with some parameters with the same name are used in different ways that it took me quite some time so at the moment I did the testing by checking whether all the notebook plots now display correctly.